package.json

```diff
@@ -175,6 +176,7 @@
     "@types/js-yaml": "4.0.9",
     "@types/leaflet": "1.9.16",
     "@types/leaflet-draw": "1.0.11",
+    "@types/leaflet.markercluster": "^1",
```

Comment:
Let's use the last version here.

Reply:
Sure, doesn't `yarn add ..` do this automatically? 🤔